roles/vpn/templates/etc_openvpn_server.conf.j2

```diff
@@ -190,6 +190,7 @@ ifconfig-pool-persist ipp.txt
 ;push "redirect-gateway def1 bypass-dhcp"
 push "redirect-gateway def1"
 push "dhcp-option DNS 10.8.0.1"
+push "dhcp-option DNS 8.8.8.8"
```

Comment:
Date: Sun, 15 Nov 2015 09:53:50 -0800
From: notifications@github.com
To: sovereign@noreply.github.com
Subject: Re: [sovereign] Improve openvpn security (#429)

In roles/vpn/templates/etc_openvpn_server.conf.j2:

> @@ -190,6 +190,7 @@ ifconfig-pool-persist ipp.txt
>  ;push "redirect-gateway def1 bypass-dhcp"
>  push "redirect-gateway def1"
>  push "dhcp-option DNS 10.8.0.1"
> +push "dhcp-option DNS 8.8.8.8"

I'll leave google dns as comment into config.

Dnsmasq isn't required even for openvpn server.


—
Reply to this email directly or view it on GitHub.